### PR TITLE
Getting version number of installed library

### PR DESCRIPTION
**Pull request by dwalton76**
For issue #200

### CODE REVIEW

File: ev3dev/core.py [hunk 1]
```diff
@@ -61,6 +61,9 @@
 INPUT_AUTO = ''
 OUTPUT_AUTO = ''
 
+# Keep the __version__ in sync with the package version in debian/changelog
+__version___ = '0.7.0'
```
Comment (dlech) [inline ev3dev/core.py]: Is the triple underscore intentional?

Surely there is a way to do this automatically, e.g. with setuptools?

Comment (dwalton76) [inline ev3dev/core.py]: triple underscore...good eye that is a typo, will fix.

There is probably a way to do this automatically but we aren't doing releases at a frantic pace so for now I think this is fine.  We can automate it down the road.

**Comment by WasabiFan** [comment]
Nice! I should add a note about this in the README.

P.S. I'd recommend using the "squash and merge" option when merging PRs to squash all the commits into a single one. It's nice in cases like this where there's an extra commit just to fix a typo.

**Comment by dwalton76** [comment]
Ah did not notice that github had a "squash and merge" option....slick.

**Pull request merged**